qemu/tests/cfg/virtio_mem_ignore_shared.cfg

```diff
@@ -0,0 +1,35 @@
+- migration_virtio_mem_ignore_shared:
```

Comment:
The case name is better consistent with the file name, other parts are ok for me :)

Reply:
Updated, thanks !